frontend/src/budgetSimulation/budgetSimulation.js

```diff
@@ -33,16 +33,23 @@ class Citizen extends React.Component {
     render() {
         const statistic = (
             <Popover id={"popover-basic"}>
-                <Popover.Title as={"h3"}> {this.props.data.name} </Popover.Title>
                 <Popover.Content>
-                    {Object.keys(this.props.data.traits).map((trait, key) =>
-                        (<div key={key}>
-                            <strong>
-                                {trait.split("_").join(" ")}: &nbsp;
-                            </strong>
-                            {this.props.data.traits[trait] ? "True" : "False"}
-                            <br/>
-                        </div>)
+                    Citizen {this.props.data.name} &nbsp;
+                    {Object.keys(this.props.data.traits).map((trait, i) => {
+                        console.log(Object.keys(this.props.data.traits).length);
```

Comment:
This block is complicated enough that I'd suggest factoring out as a little helper function, so you end up with something cleaner inside the JSX.

Reply:
factored out into a helper function!

Reply:
> factored out into a helper function!

Hearing words like that always makes my heart go a-flutter!